packages/invoices/src/InboundShipment/api/hooks.ts

```diff
@@ -145,7 +149,7 @@ export const useSaveInboundLines = () => {
   const queryClient = useQueryClient();
   const invoiceNumber = useInboundNumber();
   const api = useInboundApi();
-  return useMutation(api.upsertLines, {
+  return useMutation(api.updateLines, {
```

Comment:
Whats a word for update, create or delete 🤔

Reply:
mutate? 😆 
`doStuffTo` 🤷

Reply:
`api.doStuff(..)`

The ultimate "I give up" 😁